package.json

```diff
@@ -48,8 +48,9 @@
     "version-packages-next": "npx lerna version \"0.0.0-next-$(git rev-parse HEAD)\" --exact --private --no-git-tag-version --force-publish --no-push --yes",
     "publish-packages-next": "npx lerna publish from-package --no-private --dist-tag next --pre-dist-tag next --yes",
     "prepare": "husky install",
-    "snyk-test": "node scripts/snyk-test.js",
-    "pregenerate-vulnerability-report": "npm run compile -w packages/compass && npm run snyk-test",
+    "snyk-test": "concurrently -r \"compass-scripts snyk-test\" \"npx lerna exec 'compass-scripts snyk-test'\"",
```

Comment:
Can we rather add a `Promise.all` here? https://github.com/mongodb-js/compass/pull/5610/files#diff-69ca8c92edfd3d28153d8dbeb3cebd671ac4ee3e80bb425983d2f7ce0ecec3d4L80

Is quite nice to keep this in one script so we can do changes in `mongosh` with one copy paste. We can also add it `snyk-test` and `snyk-report` to sbom-tools so it is easier to share, but since there are plans to centralize snyk (at least for SSDLC) i'm not sure is worth investing too much on it

Reply:
I'll open a ticket to move it to mongosh, wasn't obvious to me that we can't change internal scripts because they are used somewhere else